### PR TITLE
deposit: arxiv fft get refactor

### PR DESCRIPTION
**Pull request by pedrogaudencio**

### CODE REVIEW

**Comment by jalavik** [comment]
Looks good. Will give it a test.

**Comment by pedrogaudencio** [comment]
It was just so we don't lose the helper function... I believe it's clearer now even though we're not harvesting the arXiv pdf. ;)

File: inspire/modules/deposit/tasks.py [hunk 1]
```diff
@@ -56,21 +64,23 @@ def arxiv_fft_get(obj, eng):
             dir=CFG_TMPSHAREDDIR,
         )
 
-        os.write(arxiv_file, arxiv_pdf.content)
-        os.close(arxiv_file)
```
Comment (jalavik) [inline inspire/modules/deposit/tasks.py]: @pedrogaudencio I think you still need to call `os.close` to properly close the `arxiv_file` descriptor. See https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp

**Comment by jalavik** [comment]
Apart from the above comment, it looks fine. I will merge after fix and rebase + re-push.

**Comment by pedrogaudencio** [comment]
Fixed, rebased and re-pushed ;)

**Comment by jalavik** [comment]
Thanks!

**Pull request merged**